src/main/java/dev/brus/downstream/updater/App.java

```diff
@@ -639,4 +645,21 @@ private static void loadRevertingChain(Map.Entry<GitCommit, String> revertingCom
          revertingChain.add(revertingCommitEntry.getValue());
       }
    }
-}
\ No newline at end of file
+
+   private static void synchronizeIssues(IssueManager upstreamIssueManager, DownstreamIssueManager downstreamIssueManager) throws Exception {
+      for (Issue upstreamIssue : upstreamIssueManager.getIssues()) {
+         if (downstreamIssueManager.getIssues().stream().noneMatch(issue -> issue.getIssues().contains(upstreamIssue.getKey()))) {
+            logger.info("Creating downstream issue for upstream issue " + upstreamIssue.getKey());
+
+            Issue downstreamIssue = ((RedHatJiraIssueManager) downstreamIssueManager).createIssue(
+               upstreamIssue.getSummary(),
+               upstreamIssue.getDescription(),
+               upstreamIssue.getType(),
+               upstreamIssue.getAssignee(),
+               "",
+               upstreamIssue.getLabels());
```

Comment:
The upstream/downstream issue types and labels could not match, see the method [cloneUpstreamIssue](https://github.com/brusdev/downstream-updater/blob/main/src/main/java/dev/brus/downstream/updater/CommitProcessor.java#L1105)

Reply:
The current GithubIssueManager only detects 2 issues types: Bug and Enhancement
https://github.com/brusdev/downstream-updater/blob/main/src/main/java/dev/brus/downstream/updater/issue/GithubIssueManager.java#L238

In general each issue manager instance could have different issue types. We could try to detect only the ones we want clone and convert them, i.e. Bug and Enhancement